insights/parsers/lsblk.py

```diff
@@ -264,9 +299,15 @@ def parse_content(self, content):
             def str2bool(s):
                 return bool(int(s))
 
+            if 'TYPE' not in d:
+                raise ParseException(
+                    "TYPE not found in LsBlockPairs line '{l}'".format(l=line)
+                )
+
             for original, replace, transform in [("RM", "REMOVABLE", str2bool),
                                                  ("RO", "READ_ONLY", str2bool)]:
                 if original in d:
                     d[replace] = transform(d[original]) if transform else d[original]
                     del d[original]
             self.rows.append(BlockDevice(d))
+        self.device_data = {dev.name: dev for dev in self.rows}
```

Comment:
@PaulWay you'll need to revert to the Python 2.6 version of this dictionary comprehension to fix the build error.

Reply:
Dangit, I saw the constructor for d on line 297 and thought "yep, no dictionary constructors here!" and completely forgot about my own copy and paste.